code/game/objects/items/trader.dm

```diff
@@ -350,6 +350,7 @@
 			last_donut = world.timeofday
 
 //Security Skirt spritework is coutesy of TG, CC BY-SA 3.0 license
+/*
```

Comment:
Do not comment like this, it fucks up conflict solving. Also why not just remove the code? And the icons?

Reply:
It messes up conflict solving?
2) I am lazy and our sins must be exposed, history repeats itself and all that

Reply:
Anyway I am going to remove it from code BRB